TypeCobol/Compiler/Parser/CodeElementBuilder/CobolCodeElementBuilder.cs

```diff
@@ -18,7 +18,8 @@ namespace TypeCobol.Compiler.Parser
 	/// <summary>Builds a CodeElement object while visiting its parse tree.</summary>
 	internal partial class CodeElementBuilder: CodeElementsBaseListener {
 
-		private ParserRuleContext Context;
+	    private bool DebuggingMode { get; set; }
```

Comment:
Would be clearer if renamed to 'IsDebuggingModeEnabled'

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/8adf8c367cfdcd4aaa32b91135b29bd72e31b660